cmake/IncludeSource.cpp.in

```diff
@@ -0,0 +1 @@
+#include "@IMPL@"
```

Comment:
Nit (This way we would not need to build separate compilation rules for every CPU capability)
```suggestion
#define CPU_CAPABILITY @CPU_CAPABILITY@
#include "@IMPL@"
```

Reply:
I had considered that, but we still need to pass the vectorization flags like `-march=avx2` so I think It makes sense to define everything `CPU_CAPABILITY` related on the command line.